service/src/config.rs

```diff
@@ -75,7 +75,7 @@ pub struct Config {
         short,
         long,
         env,
-        default_value_t = LevelFilter::Warn,
```

Comment:
Changing this as I always wind up changing it when developing locally. We want to be able to see `Info` level logs as a default.

Reply:
This is intentionally set to Warn as the default so we don't unnecessarily expose details for a production run, and then I always run the backend with the following command line: `cargo run  -- -l ERROR -d postgres://refactor:password@localhost:5432/refactor_platform --allowed-origins="http://localhost:3000"`

Reply:
Discussed and agreed that Warn makes sense even for production runtime.